staging/src/k8s.io/apiserver/pkg/server/options/authorization.go

```diff
@@ -41,6 +42,9 @@ type DelegatingAuthorizationOptions struct {
 	// RemoteKubeConfigFile is the file to use to connect to a "normal" kube API server which hosts the
 	// SubjectAccessReview.authorization.k8s.io endpoint for checking tokens.
 	RemoteKubeConfigFile string
+	// RemoteKubeConfigFileOptional is specifying whether not specifying the kubeconfig or
+	// a missing in-cluster config will be fatal.
+	RemoteKubeConfigFileOptional bool
```

Comment:
if we don’t add it to the flagset，where can we set its value?

Reply:
the component using the option struct sets it.

Reply:
i thought that every field in option struct should be added to flagset

Reply:
No, why? We have many which are not.

Reply:
look into SecureServing as an example.